### PR TITLE
Added seed management command

### PR DESCRIPTION
**Pull request by g8tor**
@childers Here is the pull request we just discussed.

In short, when merged the following will be the result

1. Update the package.son file to reflect the new package names and installation location
2. Add a migration to blast so that we can automatically seed the blast_sequencetype table
3. Add the seed management command to seed

Thanks

-Vern

### CODE REVIEW

File: app/management/commands/seed.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from django.core import management
+from django.core.management.base import BaseCommand, CommandError
```
Comment (hsiaoyi0504) [inline app/management/commands/seed.py]: Codacy Issue found: ['django.core.management.base.CommandError' imported but unused](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4526914)

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- blast/migrations/0009_auto_20191105_1631.py  1
- app/management/commands/seed.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/genomics-workspace/genomics-workspace/pullRequest?prid=4526914&bid=15242138)

File: app/management/commands/seed.py (hunk 1 above)
Comment (hsiaoyi0504) [inline app/management/commands/seed.py]: Codacy Issue found: [Unused CommandError imported from django.core.management.base](https://app.codacy.com/app/NAL-i5K/genomics-workspace/pullRequest?prid=4526914)

Comment (g8tor) [reply app/management/commands/seed.py]: Removed Unused CommandError

**Comment by hsiaoyi0504** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- blast/migrations/0009_auto_20191105_1631.py  1
- app/management/commands/seed.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/genomics-workspace/genomics-workspace/pullRequest?prid=4526914&bid=15242138)

**Pull request merged**